### PR TITLE
Cuts down on hairstyle power creep(shaver port)

### PR DESCRIPTION
**Pull request by JustSumBody**
nerfs #13906
Fun fact: The sprite for this was in the .dmi this whole time. Doesn't seem to have an inhand though, not sure if that's kicking around somewhere or someone wants to take a crack at it
Basically: Aim at head or face to shave. Takes 5 seconds on other people, half a second on yourself. 
image

:cl:
 * rscadd: Adds an electric shaver to the autolathe.

### CODE REVIEW

File: code/game/objects/items/weapons/cosmetics.dm [hunk 1]
```diff
@@ -260,3 +260,75 @@
...
+				return
+			if(H.f_style == "Shaved")
+				to_chat(user,"<span class='warning'>Already clean-shaven!</span>")
+				return
+
+			if(H == user) //shaving yourself
+				user.visible_message("[user] starts to shave their facial hair with [src].", \
+									 "<span class='notice'>You take a moment to shave your facial hair with [src]...</span>")
+				if(do_after(user, H, 5))
+					user.visible_message("[user] shaves his facial hair clean with [src].", \
```
Comment (N3X15) [inline code/game/objects/items/weapons/cosmetics.dm]: >his

YOU DON'T KNOW ME

And before you ask, yes, Character Setup lets females have facial hair.

File: code/game/objects/items/weapons/cosmetics.dm [hunk 2]
```diff
@@ -260,3 +260,75 @@
...
+					user.visible_message("<span class='warning'>[user] shaves off [H]'s facial hair with [src].</span>", \
+										 "<span class='notice'>You shave [H]'s facial hair clean off.</span>")
+					shave(H, location)
+
+		else if(location == LIMB_HEAD)
+			if(H.check_body_part_coverage(HEAD))
+				to_chat(user,"<span class='warning'>The headgear is in the way!</span>")
+				return
+			if(H.h_style == "Bald" || H.h_style == "Balding Hair" || H.h_style == "Skinhead")
+				to_chat(user,"<span class='warning'>There is not enough hair left to shave!</span>")
```
Comment (N3X15) [inline code/game/objects/items/weapons/cosmetics.dm]: Balding has hair that can be shaved.

File: code/game/objects/items/weapons/cosmetics.dm [hunk 3]
```diff
@@ -260,3 +260,75 @@
...
+				return
+
+			if(H == user) //shaving yourself
+				user.visible_message("[user] starts to shave their head with [src].", \
+									 "<span class='notice'>You start to shave your head with [src]...</span>")
+				if(do_after(user, H, 5))
+					user.visible_message("[user] shaves his head with [src].", \
```
Comment (N3X15) [inline code/game/objects/items/weapons/cosmetics.dm]: Needs more `\his`

**Review by N3X15 — CHANGES_REQUESTED** [review]

**Comment by 9600bauds** [comment]
Leave one of these in the bathrooms somewhere, have more orderable from C*rgo or an Autolathe I guess

**Review by N3X15 — APPROVED** [review]

**Comment by TheFriggenSawbones** [comment]
bARBERS
'WHEN

**Comment by gbasood** [comment]
@TheFriggenSawbones wyci
not like it's harder than skill capes

**Comment by Lokiamis** [comment]
There should be one for the warden to use on prisoners, as is tradition.

**Comment by Probe1** [comment]
Add it to the Autolathe.  A crate in cargos silly.

File: code/game/machinery/autolathe.dm [hunk 4]
```diff
@@ -102,6 +102,7 @@
 		new /obj/item/clothing/head/welding(), \
 		new /obj/item/device/taperecorder(), \
 		new /obj/item/weapon/chisel(), \
+		new /obj/item/weapon/razor, \
```
Comment (9600bauds) [inline code/game/machinery/autolathe.dm]: does this really not need the ()

Comment (JustSumBody) [reply code/game/machinery/autolathe.dm]: Wew, it worked without it. That's interesting

**Comment by JustSumBody** [comment]
I detest mapping, autolathe for now. If someone else wants to go for it.

**Comment by Shadowmech88** [comment]
You could avoid mapping if you put it in the code for the warden's locker.

**Comment by Ephx1** [comment]
This could be cool on the service borg module

**Comment by Probe1** [comment]
Something interesting for service borgs?  NO HAVE MORE DICE.

**Pull request merged**